guides/v2.2/comp-mgr/extens-man/extensman-main-pg.md

```diff
@@ -29,11 +29,8 @@ To start the {% glossarytooltip 55774db9-bf9d-40f3-83db-b10cc5ae3b68 %}Extension
 6.	Enter your authentication keys in the provided fields.
 2.	Click **Submit**.
 
-<div class="bs-callout bs-callout-warning" markdown="1">
-You must use the same authentication keys you used to install the Magento software. For example, you *cannot* use {{site.data.var.ce}} authentication keys to update or upgrade {{site.data.var.ee}} or vice versa. You also *cannot* use:
-* Another user's authentication keys
-* [Shared account](http://docs.magento.com/m2/ce/user_guide/magento/magento-account-share.html){: target="_blank"} authentication keys
-</div>
+{:.bs-callout .bs-callout-warning}
+You must use the same authentication keys you used to install the Magento software. For example, you *cannot* use {{site.data.var.ce}} authentication keys to update or upgrade {{site.data.var.ee}} or vice versa. You also *cannot* use another user's authentication keys or [Shared account](http://docs.magento.com/m2/ce/user_guide/magento/magento-account-share.html){: target="_blank"} authentication keys
```

Comment:
Delete the extra space between the colon and attribute or delete the attribute completely:

>`{: target="_blank"}`